charts/kube-otel-stack/values.yaml

```diff
@@ -154,12 +183,31 @@ metricsCollector:
         detectors: [env]
         timeout: 2s
         override: false
+      k8sattributes:
+        passthrough: false
+        pod_association:
+          - sources:
+              - from: resource_attribute
+                name: k8s.pod.name
+        extract:
+          metadata:
+            - k8s.cluster.name
+            - k8s.namespace.name
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.node.name
+            - k8s.pod.start_time
+            - container.image.tag
+            - container.image.name
```

Comment:
I think you can also add `k8s.deployment.name`, `k8s.replicaset.name`, `k8s.statefulset.name`, `k8s.daemonset.name`. I have observed all these attributes when using the appropriate workload mode. https://app.lightstep.com/dev-mohosman/notebooks/workload-type/gv8gDRkf

I haven't been able to get `k8s.cronjob.name` to showup but that's probably me not setting up the cronjob correctly, rather than it actually being missing

Reply:
done, thanks!